apps/web-client/src/routes/history/date/+page.svelte

```diff
@@ -137,50 +117,54 @@
-			{#if !$allEntriesList || !$allEntriesList.bookList || !$allEntriesList.bookList.length}
+			{#if !$dailySummary?.bookList?.length}
 				<!-- Start entity list placeholder -->
 				<PlaceholderBox title="No Books on that date" description="Try selecting a different date." class="center-absolute" />
 				<!-- End entity list placeholder -->
 			{:else}
 				<!-- Start entity list -->
 				<div class="flex flex-row text-sm">
 					<div class="badge badge-green m-2 p-2 font-bold">
-						Inbound Book Count: {$allEntriesList.stats.totalInboundBookCount}
+						Inbound Book Count: {$dailySummary.stats.totalInboundBookCount}
 					</div>
 					<div class="badge badge-green m-2 p-2 font-bold">
-						Inbound Cover Price: {$allEntriesList.stats.totalInboundCoverPrice.toFixed(2)}
+						Inbound Cover Price: {$dailySummary.stats.totalInboundCoverPrice.toFixed(2)}
 					</div>
 					<div class="badge badge-green m-2 p-2 font-bold">
-						Inbound Discounted Price : {$allEntriesList.stats.totalInboundDiscountedPrice.toFixed(2)}
+						Inbound Discounted Price : {$dailySummary.stats.totalInboundDiscountedPrice.toFixed(2)}
 					</div>
 				</div>
 				<div class="flex flex-row text-sm">
 					<div class="badge badge-red m-2 p-2 font-bold">
-						Outbound Book Count: {$allEntriesList.stats.totalOutboundBookCount}
+						Outbound Book Count: {$dailySummary.stats.totalOutboundBookCount}
 					</div>
 					<div class="badge badge-red m-2 p-2 font-bold">
-						Outbound Cover Price: {$allEntriesList.stats.totalOutboundCoverPrice.toFixed(2)}
+						Outbound Cover Price: {$dailySummary.stats.totalOutboundCoverPrice.toFixed(2)}
 					</div>
 					<div class="badge badge-red m-2 p-2 font-bold">
-						Outbound Discounted Price : {$allEntriesList.stats.totalOutboundDiscountedPrice.toFixed(2)}
+						Outbound Discounted Price : {$dailySummary.stats.totalOutboundDiscountedPrice.toFixed(2)}
 					</div>
 				</div>
 
-				{#each $allEntriesList.bookList as entry}
+				{#each $dailySummary.bookList as entry}
 					{@const title = entry.title}
 					{@const quantity = entry.quantity}
 					{@const warehouseName = entry.warehouseName}
-					{@const committedAt = entry.committedAt}
+					{@const committedAt = entry.date}
 					{@const noteType = entry.noteType}
+					{@const noteName = entry.noteDisplayName}
+					{@const noteId = entry.noteId}
 
 					<li class="entity-list-row grid grid-flow-col grid-cols-12 items-center">
 						<div class="max-w-1/2 col-span-10 row-span-1 w-full xs:col-span-6 lg:row-span-2">
 							<p class="entity-list-text-lg text-gray-900">{title}</p>
 
 							<div class="flex items-center">
 								<Library class="mr-1 text-gray-700" size={20} />
-								<span class="entity-list-text-sm text-gray-500">{quantity} books -</span>
-								<span class="entity-list-text-sm text-gray-500"> {warehouseName}</span>
+								<span class="entity-list-text-sm text-gray-500">{quantity} books - </span>
```

Comment:
Only comment is that text here could use a couple additional spaces or a few less "(" ")"